File: app/src/main/java/com/example/loginapp/MainActivity.java
package com.example.loginapp;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    TextView login;
    TextView signup;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        login = findViewById(R.id.signIn);
        signup = findViewById(R.id.signUp);

        login.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent loginIntent = new Intent(MainActivity.this, Login.class);
                startActivity(loginIntent);
            }
        });

        signup.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent signUpIntent = new Intent(MainActivity.this, Register.class);
                startActivity(signUpIntent);
            }
        });
    }


}

File: app/src/main/java/com/example/loginapp/Activities/MainActivity.kt
package com.example.loginapp.Activities

import android.content.Intent
import android.os.Bundle
import android.view.View
import android.widget.TextView
import android.widget.Toast
import androidx.appcompat.app.AppCompatActivity
import com.example.loginapp.R

class MainActivity : AppCompatActivity(), View.OnClickListener {
    private lateinit var signIn: TextView
    private lateinit var signUp: TextView
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        initViews()
        initListeners()
    }

    private fun initViews() {
        signIn = findViewById(R.id.signIn)
        signUp = findViewById(R.id.signUp)
    }

    override fun onClick(v: View) {
        when (v.id) {
            R.id.signUp -> startActivity(Intent(this@MainActivity, Register::class.java))
            R.id.signIn -> startActivity(Intent(this@MainActivity, Login::class.java))
        }
    }

    private fun initListeners() {
        signIn.setOnClickListener(this)
        signUp.setOnClickListener(this)
    }

    override fun onBackPressed() {
        super.onBackPressed()
        return
    }
}

